portfolio/src/main/webapp/script.js

```diff
@@ -454,5 +462,62 @@ function userMaps() {
     createUserMap();
 }
 
+// Callback that creates and populates a data table,
+// instantiates the pie chart, passes in the data and
+// draws it.
+function drawPieChart() {
+
+    // Create the data table.
+    var data = new google.visualization.DataTable();
+    data.addColumn('string', 'Living Status');
+    data.addColumn('number', 'Percentages');
+    data.addRows([
+        ['Children with homes', 31.4],
+        ['Children living on streets', 56.9],
+        ['Children living both home and streets', 11.8],
+    ]);
+
+    // Set chart options
+    var options = {'title':'Zimbabwe children Living Status',
+                    backgroundColor: 'transparent',
+                    'width':500,
+                    'height':450};
+
+    // Instantiate and draw our chart, passing in some options.
+    var chart = new google.visualization.PieChart(document.getElementById('chart-one'));
+    chart.draw(data, options);
+}
 
-
+function drawAfricaMap() {
```

Comment:
Wouldn't it be better if you use Google Maps API for this? Or if you like the visual effects of this - that's fine as well

Reply:
The chat api also calls the maps api.